plugins/login-resources/src/components/LoginApp.svelte

```diff
@@ -88,14 +88,15 @@
       />
     </picture>
 
-    <div
+    <a
+      href="https://bold.ru"
```

Comment:
This should be configurable with environment variable, and passed from front service using config.

Reply:
Thanks! I added a variable to the config

Reply:
@haiodo can you help me please with uitests? 
Unfortunately, I don’t understand at all what the problem is with it

Reply:
Please try to rebase at least.

Reply:
And you need to pass LOGOWAY_URL variable to tests/docker-compose.yaml front container.

Reply:
Thank you
I'm added variable to compose and tried to to pull all changes but problem with pipeline is not solved